doc/nrf/config_and_build/bootloaders/index.rst

```diff
@@ -51,12 +51,17 @@ You can find an overview of currently supported bootloaders in the table below:
      - Yes
      - :doc:`See imgtool <mcuboot:imgtool>`
      - No
-     - Yes
+     - Yes `*`
      - Yes
      - :ref:`Monotonic (HW) <bootloader_monotonic_counter>`, :ref:`Semantic (SW) <ug_fw_update_image_versions_mcuboot>`
      - Image swap - single primary
        Dual-slot direct-xip
 
+The table symbol can be read as follows:
+
+``*`` SMP with MCUboot is only available for Bluetooth® Low Energy and serial UART.
+(See the :ref:`ug_fw_update` page to learn about other DFU methods.)
```

Comment:
These are notes for MCUboot? We do not support bluetooth in MCUboot dfu, we only support it in applications.
We support CDC USB as uart device in MCUboot.

Reply:
@hellesvik-nordic  can you please take a look at this? Maybe I haven't worded something correctly.

Reply:
Oh, is that what this column means? What DFU method is supported inside the bootloader?
That is not how I read it at all, but now that you say it that makes sense.
If I misunderstand this, chances are others will as well.
Then the question would be: How can we make it clear that the table talks about DFU from the bootloader itself?

Reply:
To elaborate:
How I read it before: "When you use this bootloader, the device can be DFUed via SMP"
How I read it now: "The bootloader has SMP functionality integrated in the bootloader image"

Reply:
Hm, maybe the column can just be renamed to something like, "SMP capabilities"?
![image](https://github.com/nrfconnect/sdk-nrf/assets/51916561/b7c678e7-2f0b-45dc-9a67-32ccc8aea10e)

Reply:
SMP Capabilities still could be read as "NSIB does not support DFU". 
What are we trying to get across with this column? What do readers gain from it?

Reply:
I think column "smp updates" is about application capabilities, not bootloader capabilities. We rather need another column for describe bootloader build in recovery-DFU capabilities like serial recovery or USB-DFU class available in MCUboot.

Maybe the column title should be renamed to something like "SMP updates by the application"?

Reply:
I have attempted to address these changes - updated the table for Bootloaders, and added some information in the "DFU" index page. Please take a look.